datahub-graphql-core/src/main/resources/ingestion.graphql

```diff
@@ -66,6 +66,12 @@ extend type Mutation {
   Cancel a running execution request, provided the urn of the original execution request
   """
   cancelIngestionExecutionRequest(input: CancelIngestionExecutionRequestInput!): String
+
+  """
+  Create a request to execute a test ingestion connection job
+  input: Input required for creating a test connection request
+  """
+  createTestConnectionRequest(input: CreateTestConnectionRequestInput!): String
```

Comment:
I'm a bit confused - why is the return type here a String? Would a Boolean not be better (connection works or does not)? I know we return Strings from some of our GraphQL endpoints but I am personally not a huge fan

Reply:
So here we return an URN for the task. 

Then we poll for the result of that urn... It's not blocking